src/features/transfer/TransfersDetailsModal.tsx

```diff
@@ -80,9 +81,11 @@ export function TransfersDetailsModal({
 
   const isAccountReady = !!account?.isReady;
   const connectorName = account?.connectorName || 'wallet';
-  const token = getToken(tokenCaip19Id);
+  // TODO confirm works for native tokens? Should findToken create for ALL unfound natives?
```

Comment:
> Should findToken create for ALL unfound natives?

what does this mean?

Reply:
Sorry, this was a note for myself during development that slipped through. It's done, I'll remove the comment